### PR TITLE
MC-1107 updating documentation so that it uses mc-start.sh instead of java -jar

### PR DESCRIPTION
**Pull request by erosb**

### CODE REVIEW

File: docs/modules/ROOT/pages/configuring.adoc [hunk 1]
```diff
@@ -36,21 +36,19 @@ This also means no new license can be stored when the property is used.
 [[starting-with-an-extra-classpath]]
 == Providing an Extra Classpath
 
+Management Center is distributed with an empty `bin/user-lib/` directory. You can put your libraries into this library to use them
```
Comment (zhmaeff) [inline docs/modules/ROOT/pages/configuring.adoc:39]:
```suggestion
Management Center is distributed with an empty `bin/user-lib/` directory. You can put your libraries into this directory to use them
```

File: docs/modules/ROOT/pages/configuring.adoc [hunk 2]
```diff
@@ -36,21 +36,19 @@ This also means no new license can be stored when the property is used.
...
+from Management Center, so any jar file in this directory will be added to the classpath.
+
 You can also start the Management Center with an extra classpath
 entry (for example, when using JAAS authentication) by using the
 following command:
 
 [source,bash,subs="attributes+"]
 ----
-java -cp "hazelcast-management-center-{full-version}.jar:/path/to/an/extra.jar" com.hazelcast.webmonitor.Launcher
+export MC_CLASSPATH="/path/to/an/extra.jar"
```
Comment (zhmaeff) [inline docs/modules/ROOT/pages/configuring.adoc:48]: Shouldn't we still describe 2 options (for Win and others) I assume there will be a difference in the classpath separator?

File: docs/modules/ROOT/pages/configuring.adoc (hunk 1 above)
Comment (erosb) [reply docs/modules/ROOT/pages/configuring.adoc]: nice catch, thanks

File: docs/modules/ROOT/pages/configuring.adoc (hunk 2 above)
Comment (erosb) [reply docs/modules/ROOT/pages/configuring.adoc:48]: Added separate tabs for windows and unix everywherem where an `mc-start.sh` example is used.

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 3]
```diff
@@ -61,18 +61,27 @@ For all the starting options you can preconfigure a cluster in Management Center
...
 === Using the Command Line
 
 Extract the Hazelcast or Management Center package.
 The extracted directory, i.e., `hazelcast-management-center-{full-version}`,
-contains the `hazelcast-management-center-{full-version}.jar` file.
+contains a `bin/mc-start.sh` (or `bin/mc-start.cmd`) file.
 
 You can start this file directly from the command line using the following command:
 
+On Unix systems:
```
Comment (zhmaeff) [inline docs/modules/ROOT/pages/getting-started.adoc:73]: nit: tabs can be used here

**Review by zhmaeff — APPROVED** [review]
Awesome! thanks

**Pull request merged**